.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Description
+Please include a summary of change(s) and/or link to associated [issue](https://github.com/SAP/open-ux-tools/issues)
+
+## Type of change
+- [ ] Bug (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds a new feature)
+- [ ] Breaking change (Bug or New feature that would cause existing functionality/consumers to not work as expected)
+- [ ] Non-Breaking chores (Changes to tools, libraries, build process, documentation, etc)
+- [ ] None of the above (Reviewers might ask for more clarification)
+
+## How have you tested?
+Make it easier for reviewer by describing the tests you have done to verify your changes and provide clear instructions for reviewers to reproduce.
+
+## Checklist:
+- [ ] The code conforms to the [general development principles](https://github.com/SAP/open-ux-tools/blob/main/docs/Guidelines.md)
+- [ ] Supplied as many details as possible on this change
+- [ ] The code is easy to read and maintainable by others
```

Comment:
Just realized that we don't much about this in the dev guidelines. We probably need a section there

Reply:
`we don't have much about this in the dev guidelines. `

Yes I noticed that too. May be after this pr goes through we can start adding some more guidelines there? This template should not change